### ISSUE TITLE
Disabling label output in the SelectInput component as in the TextInput component

### ISSUE THREAD

**Issue by MaxAlex666** [marmelab/react-admin#3657]
react-admin: 2.9.6
In the TextInput component, you can completely hide the line with the label using the attribute label={false}.
In the SelectInput component, you can only hide the label text with the label="" attribute, but the empty string still remains. 
Using a small change, it is possible to implement a similar behavior for SelectInput:

```
SelectInput.propTypes = {
    label: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
    ...
```     
```
        return (
            <ResettableTextField
                select
                margin="normal"
                value={this.state.value}
                label={ label === false ? null :
                    <FieldTitle
                ...  
```

**Comment by rahulmalhotra** [comment]
Hey @Kmaschta I would like to work on this issue, can you assign it to me ?

**Comment by Kmaschta** [comment]
@rahulmalhotra Any contribution is welcome! Thanks
We do not assign issues, you can open a PR and add `Closes #3657` in the description, the issue will be linked automatically.

**Comment by rahulmalhotra** [comment]
Got that. Working on the same. Thanks..!!

**Comment by fzaninotto** [comment]
Fixed by #3806
